.github/workflows/typing_conformance_comment.yaml

```diff
@@ -61,7 +61,7 @@ jobs:
           # subsequent runs
           echo '<!-- generated-comment typing_conformance_diagnostics_diff -->' >> comment.txt
 
-          echo '## Diagnostic diff on typing conformance tests' >> comment.txt
+          echo '## Diagnostic diff on [typing conformance tests](https://github.com/python/typing/tree/main/conformance)' >> comment.txt
```

Comment:
The workflow uses a specific commit which should be used here instead of `main`:

https://github.com/astral-sh/ruff/blob/808c94d509e1e3f3d22218926d08c98381ede7eb/.github/workflows/typing_conformance.yaml#L43-L43

Reply:
No problem!